pkg/errorchecker/errorchecker.go

```diff
@@ -0,0 +1,53 @@
+package errorchecker
+
+import (
+	"fmt"
+	"regexp"
+
+	"gopkg.in/check.v1"
+)
+
+type errorMatchesChecker struct {
+	*check.CheckerInfo
+}
+
+var ErrorMessageMatcher check.Checker = errorMatchesChecker{
+	&check.CheckerInfo{Name: "ErrorMatches", Params: []string{"value", "regex"}},
+}
+
+func (checker errorMatchesChecker) Check(
```

Comment:
Please add a comment on the exported functions

Reply:
Done